integrations/telegram/src/misc/utils.ts

```diff
@@ -0,0 +1,61 @@
+import { axios } from '@botpress/client'
+import { PICTURE_LIMIT_SIZE } from 'src/const'
+import { Telegraf } from 'telegraf'
+import { PhotoSize, User } from 'telegraf/typings/core/types/typegram'
+
+export const getUserNameFromTelegramUser = (telegramUser: User) => {
+  if (telegramUser.last_name) {
+    return `${telegramUser.first_name} ${telegramUser.last_name}`
+  } else if (telegramUser.username) {
+    return `${telegramUser.first_name} (username: ${telegramUser.username})`
+  }
+  return `${telegramUser.first_name} (id: ${telegramUser.id})`
+}
+
+const getDataUriFromImgHref = async (imgHref: string): Promise<string> => {
+  const fileExtension = imgHref.substring(imgHref.lastIndexOf('.') + 1)
+
+  const { data } = await axios.default.get(imgHref, { responseType: 'arraybuffer' })
+
+  const base64File = Buffer.from(data, 'binary').toString('base64')
+
+  return `data:image/${fileExtension};base64,${base64File}`
+}
+
+export const getBestPhotoSize = (photos: PhotoSize[][] | undefined): PhotoSize | null =>
```

Comment:
```suggestion
const getBestPhotoSize = (photos: PhotoSize[][] | undefined): PhotoSize | null =>
```

Reply:
I've changed the signature so it accepts only `photos: PhotoSize[]`, making it way simpler !